java/build/bazel/tests/integration/WorkspaceDriver.java

```diff
@@ -80,20 +78,20 @@ private static void unpackBazel(String version)
   }
 
   private static Command prepareUnpackBazelCommand(String version)
-    throws  IOException {
+          throws IOException {
     List<String> command = new ArrayList<String>(Arrays.asList(
-        bazelVersions.get(version).getCanonicalPath(),
-        "--output_user_root=" + tmp, "--nomaster_bazelrc",
-        "--max_idle_secs=30", "--bazelrc=/dev/null",
-        "help"));
+            bazelVersions.get(version).toString(),
```

Comment:
Do we want to normalize the path? Like get canonical? https://stackoverflow.com/questions/21552901/getting-canonical-path-from-path-in-java

Reply:
I don't think so, from what I checked the way that we're generating the path provides the absolute path that we want here. No need to convert.